src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/BookmarksModal.test.tsx

```diff
@@ -86,11 +86,14 @@ const mockState = {
         [genomeId]: geneObjectId
       }
     },
-    trackPanel: {
+    browserSidebarModal: {
```

Comment:
Not in this PR but it might be worth using [createMockBrowserState](https://github.com/Ensembl/ensembl-client/pull/726/files#diff-7d407488d78263a491106903c1e06ba1dac48dfcf4cdf13b9ec3f22817b01851R208) instead of rewriting the `mockState` here.

Reply:
I was a bit overenthusiastic and did it in this PR itself 😁

Reply:
✅